source/_integrations/leviosa_shades.markdown

```diff
@@ -0,0 +1,50 @@
+---
+title: Leviosa Motor Blinds Zone hub
+description: Instructions on how to integrate the Leviosa Zone hub into Home Assistant.
+ha_category:
+  - Cover
+ha_iot_class: Local Push
+ha_release: 2021.4
+ha_domain: leviosa_shades
+ha_codeowners:
+  - '@altersis'
+ha_config_flow: true
+ha_platforms:
+  - cover
+---
+
+This integration allows you to control [Leviosa Zone hubs](https://leviosashades.com/products/leviosa-zone) from [Leviosa Motor Shades](https://leviosashades.com/).
+
+{% include integrations/config_flow.md %}
+
+## Things to keep in mind
+
+- Each Leviosa Zone manages up to 6 groups of shades.
+- If you have more than one Leviosa Zone in your Network, Home Assistant will display a list, with one IP address for each Leviosa Zone discovered, in the same order as they were discovered. Once a Leviosa Zone is configured in Home Assistant, it will not appear in this list. Refer to the next section to find the IP address of each zone group.
+- If you have only one Leviosa Zone, then you'll be taken directly to the screen to enter the Leviosa Zone name and shade group names.
+- You can name the shade groups as you wish. They do not have to match the names already in the Leviosa App.
+- A `cover` entity for each shade group will be ready for you. An additional `cover` entity is created with the name of the zone (prefixed `all_`) to move all groups in a Leviosa Zone at the same time.
```

Comment:
The additional `all_` group/entity should not be created. It is not the responsibility of integration to provide that.

Reply:
Hi! thank you for taking a look at this. I would like to clarify that the Leviosa Zone can handle 6 shade groups, plus a special group (called Group 0 in the Leviosa Zone hub) that allows moving all Blind groups up or down. The additional shade group 'all' is just a way to expose this Group 0 functionality to the user, otherwise the would not be a way to use it. If after this explanation this appears incorrect, could you please help me to understand how the Group 0 should be exposed?

Reply:
I think it should not be exposed? They can be grouped in Home Assistant if one wishes to do so.

Reply:
I see your point, and I agree that all shade groups could be grouped; that would produce a different  user experience. I'll explain: Group 0 commands all shades to move at exactly the same time, using only one RF command over the air. All motors react to this command immediately, as they all belong to Group 0 in addition to their configured group. If I understand HASS grouping correctly, HASS would send a command to each Group in the Leviosa Zone in rapid succession, which would start raising the shades at different times, possibly with a different order each time. This happens because the Zone would send multiple RF commands over the air, one for each  shade group (RF transmission is slow, so it takes about 100 ms per command). I'm sure this behavior would lower the 'wow' factor that is expected from all blinds reacting at exactly the same time. Hope this clarifies why the 'all_' shade group is important for these blinds.